### ISSUE TITLE
Tracking flows shared with SharePoint lists/libraries

### ISSUE THREAD

**Issue by osbornecc** [microsoft/coe-starter-kit#9045]
### Does this question already exist in our backlog?

- [X] I have checked and confirm this is a new question.

### What is your question?

I love that Flows are now tracking who they are shared with.  This makes life much easier!

What we also know is that Flows can be shared with SharePoint, explicitly.
![image](https://github.com/user-attachments/assets/67cba08e-7a01-4d4a-9dd5-c3e99b3827fa)

When looking at that flow using "get flow owner role as admin" I can see they show up as "delegated" (output at the end).  Are there plans to include these shares as well?

Thank you!

```
[
  {
    "name": "c25xxxxxx",
    "id": "/providers/Microsoft.ProcessSimple/environments/xxx0ae2xxxxx/flows/7axxxxx/permissions/c25xxxxx",
    "type": "/providers/Microsoft.ProcessSimple/environments/flows/permissions",
    "properties": {
      "roleName": "Owner",
      "permissionType": "Principal",
      "principal": {
        "id": "c25xxxxx",
        "type": "User"
      }
    }
  },
  {
    "name": "DELEGATED-887xxxx",
    "id": "/providers/Microsoft.ProcessSimple/environments/xxx0ae2xxxxx/flows/7axxxxx/permissions/DELEGATED-887c0xxxxx",
    "type": "/providers/Microsoft.ProcessSimple/environments/flows/permissions",
    "properties": {
      "roleName": "CanEdit",
      "permissionType": "AuthorizationDelegate",
      "authorizationDelegate": {
        "accessDefinition": {
          "sharePointBasePermissions": [
            {
              "high": 0,
              "low": 2079
            }
          ]
        },
        "delegationSource": {
          "delegationService": "SharePoint",
          "contractVersion": "2017-12-01"
        },
        "delegatedAuthResource": {
          "resourceCollection": "https://xxxxx.sharepoint.com/teams/mytestsite",
          "resourceId": "485xxxxx"
        }
      }
    }
  }
]
```

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

4.43

### What method are you using to get inventory and telemetry?

Cloud flows

**Comment by Jenefer-Monroe** [comment]
We do not have current plans for this no, I was unaware of it! I'll put it on the backlog and see what we can do.
We are currently heads down implementing the Data Export work though so it will not be soon,

**Comment by mohamrizwa** [comment]
We haven’t heard back in a while, so we’ll proceed to close this issue for now. If the issue persists or you need further assistance, feel free to a new issue. We’re happy to help anytime. Thanks again for your engagement!

**Comment by osbornec-pnnl** [comment]
Just to clarify, there are no plans to store if a Flow is co-owned by a SharePoint list, as shown in the output above?

**Comment by mohamrizwa** [comment]
Hi @osbornecc,
The file [PowerPlatform-Compliance](https://www.microsoft365.com/chat?capabilities=interopPromise%2CsuspendOnClose%2CautoStart&client-request-id=1a6236b1-45b7-1246-9e8e-4a88ce037919&version=19.2509.34011.0#:~:text=The%20file-,PowerPlatform%2DCompliance,-outlines%20how%20co) outlines how co-ownership works in general:
     1. Flows can be shared with individuals or groups.
     2. Co-owners inherit access to connections used in the flow.
     3. There are security risks if flows use personal connections (e.g., Outlook), which may explain why SharePoint list-based ownership is not tracked.

**Comment by osbornec-pnnl** [comment]
Hi @mohamrizwa,
Our org has turned off web searching so that copilot link does not work for me.  Can you please elaborate?

**Comment by mohamrizwa** [comment]
Please proceed with upgrading to the latest version of the Center of Excellence (CoE) Starter Kit. Before starting, ensure all unmanaged layers are identified and removed to avoid deployment issues. 

For detailed guidance, refer to the below -mentioned 

 official documentation: 

 https://learn.microsoft.com/power-platform/guidance/coe/after-setup 

Feel free to reach out if you need any assistance. We're happy to help!

**Comment by mohamrizwa** [comment]
We haven’t heard back in a while, so we’ll proceed to close this issue for now. If the issue persists or you need further assistance, feel free to a new issue. We’re happy to help anytime. Thanks again for your engagement!

**Comment by mohamrizwa** [comment]
We haven’t heard back in a while, so we’ll proceed to close this issue for now. If the issue persists or you need further assistance, feel free to a new issue. We’re happy to help anytime. Thanks again for your engagement!
